torchx/workspace/docker_workspace.py

```diff
@@ -135,6 +141,9 @@ def build_workspace_and_update_role(
                     self.LABEL_VERSION: torchx.__version__,
                 },
             )
+            if cfg.get("verbose", False):
+                for line in output:
+                    log.info(line)
```

Comment:
do these lines run even when there's an error in building?

Reply:
Yes, @d4l3k that's one of the reasons to print this. See https://github.com/docker/docker-py/blob/main/docker/models/images.py#L303